websites/P/PlanetXO/presence.ts

```diff
@@ -0,0 +1,200 @@
+const presence = new Presence({
+		clientId: "1260572178068406282",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://cdn.kaneproductions.co.uk/art/PlanetXO/shortlogo.png",
+	};
+
+	if (document.location.hostname === "planetxo.uk") {
+		presenceData.startTimestamp = browsingTimestamp;
+		switch (document.URL.split("/").slice(0, -1).join("/")) {
+			case "https://planetxo.uk": {
+				presenceData.details = "Viewing the map";
+				presenceData.buttons = [
+					{
+						label: "Home",
+						url: "https://planetxo.uk",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/xoradio": {
+				presenceData.details = "Viewing XO Radio";
+				presenceData.buttons = [
+					{
+						label: "Home",
+						url: "https://planetxo.uk",
+					},
+				];
+				break;
+			}
+			case "https://planetxo.uk/panel/planetxo": {
+				presenceData.details = "Viewing the PlanetXO Global Panel";
+				presenceData.buttons = [
+					{
+						label: "Home",
+						url: "https://planetxo.uk",
+					},
```

Comment:
Buttons which lead to the home page are not allowed in the guidelines

Reply:
Alright let me fix this real fast!